src/main/java/uk/gov/hmcts/reform/pip/rules/errorhandling/GlobalExceptionHandler.java

```diff
@@ -32,4 +33,21 @@
         return ResponseEntity.status(HttpStatus.NOT_FOUND).body(exceptionResponse);
     }
 
+    /**
```

Comment:
can we neaten this up by extending a not found exception and handling on a higher level instead?

Reply:
removed the other 404 error handling

Reply:
we still need it dont we?